account-kit/java/signer-sdk/src/main/java/com/alchemy/aa/Stamper.java

```diff
@@ -44,33 +39,15 @@ public static String toJson(APIStamp apiStamp) throws JsonProcessingException {
         return ow.writeValueAsString(apiStamp);
     }
 
-    public Stamper(TEKManager tekManager) {
-        this();
-
-        this.tekManager = tekManager;
-    }
-
-    /**
-     * @param targetPrivateKeyHex
-     *            Base64 private key
-     */
-    public Stamper(@Nonnull String targetPrivateKeyHex) throws GeneralSecurityException, IOException {
-        this();
-        byte[] keysetBytes = Base64.decode(targetPrivateKeyHex);
-        KeysetHandle privateKeysetHandle = CleartextKeysetHandle.read(BinaryKeysetReader.withBytes(keysetBytes));
+    private TekManager tekManager;
```

Comment:
Unrelated to this PR, but we could move toJson at line 37 inside the APIStamp record and make it non static. Also in general, we initializing ObjectMapper once and then using it everywhere (if we can)